src/main/java/net/mcreator/ui/SplashScreen.java

```diff
@@ -59,19 +60,18 @@ public SplashScreen() {
 		logo.setBounds(24 + 8 - 4, 70, 350, 63);
 		imagePanel.add(logo);
 
-		JLabel version = new JLabel(
-				"VERSION " + (Launcher.version != null ? Launcher.version.getMajorString() : "1234.5"));
+		JLabel version = new JLabel("VERSION " + (Launcher.version != null ? Launcher.version.getMajorString() : "1234.5"));

```

Comment:
VERSION not translated here

Reply:
:x: Won't fix
L10N is loaded while SplashScreen is shown, calling L10N from here causes app crash with exceptions from L10N.
Thus all 3 strings in the SplashScreen (`splash_screen.version`, `splash_screen.snapshot` and `splash_screen.note`) become untranslatable as they are requested **BEFORE** showing it (when only preferences are initialized!).

Reply:
Woops, my bad :D